integration_test/cloudbuild-integration-test.yaml

```diff
@@ -68,6 +86,50 @@
   waitFor:
   - wait-for-adapter
 
+- name: "maven:3.6.0-jdk-8"
+  id: run-find-scu
+  volumes:
+  - name: 'vol1'
+    path: '/root/.m2'
+  args:
+  - mvn
+  - 'exec:java'
+  - '-Dexec.mainClass=org.dcm4che3.tool.findscu.FindSCU'
+  - '-Dexec.args=-c IMPORTADAPTER@${_ADAPTER_RUN_STEP}:${_ADAPTER_PORT} -L IMAGE -X --out-cat --out-file findscu-result.xml --out-dir ../../../integration_test/'
+  dir: "dcm4che/dcm4che-tool/dcm4che-tool-findscu"
+  waitFor:
+  - run-store-scu
+
+- name: "maven:3.6.0-jdk-8"
+  id: run-commitment-scu
```

Comment:
How do we know if storage commitment SCU returned a success result? Is this captured in the return value of the mvn exec command? (0 for success, non-0 for failure?) If so, can you please add a comment explaining this.

Reply:
Commitment SCU waits for n-event-report after n-action request returns success, if it doesn't get one within cloud build step timeout of 30 seconds, step fails.

In theory, Commitment SCU would actually return 0 (success), if n-action request fails (malformed commitment request or unknown destination AET), but 1st is impossible (we are using known correct client), while 2nd would be a bug in integration itself.

Reply:
Commitment SCU does dump what it receives by default. Problem is, n-event-report contains random transaction id, so I can't compare them using diff. While regex tools don't seem to work all that well on binary files.

Reply:
But, if a server failure occurs for storage commit, the integration test won't fail since an n-event-response will be sent anyway.

What if we install the dcm2xml tool (https://github.com/dcm4che/dcm4che/tree/master/dcm4che-tool/dcm4che-tool-dcm2xml) and use this to dump the XML? Then we can remove the transaction ID using a tool like sed. Will this work? I'm not sure if dcm2xml will work on a DIMSE response.

Reply:
Added actual diff check.

I forgot to bring up one issue though (mentioned it only in git comment): api does (I'm pretty sure incorrectly) return  "generic::invalid_argument: SOPClassUID is not a supported instance or study or series level attribute" on qido-rs queries used by commitment service.
Most of my testing is done with local dcm4che, where such queries work.

So current commitment test just validates that api still doesn't work.